salt/metalk8s/runc/installed.sls

```diff
@@ -3,8 +3,22 @@
 include:
   - metalk8s.repo
 
-# TODO: Add a fromrepo for offline
+Install criu:
+  pkg.installed:
+    - name: criu
+    - version: {{ repo.packages.criu.version }}
+    - fromrepo: {{ repo.packages.criu.repository }}
```

Comment:
why dont we use the same format, either `repo.packages['XX']` or `repo.packages.XX`

Reply:
Need to quote with `container-selinux' because of the dash